integrations/exe_analyzers/Dockerfile

```diff
@@ -0,0 +1,49 @@
+FROM python:3.8.4-slim
+
+ENV PROJECT_PATH /opt/deploy
+ENV LOG_PATH /var/log/intel_owl/exe_analyzers
+
+RUN mkdir -p $LOG_PATH
```

Comment:
Doesn't this directory gets created in the main IntelOwl dockerfile ? Is there a reason you added this command here ?

Reply:
Oh, I should remove it actually,Thanks for pointing out. While testing several methods to resolve it, I tried this once. Yeah, In the first build it didn't get created. From next time, It's working fine. I'll remove this line now.